CONTRIBUTING.md

```diff
@@ -7,6 +7,13 @@ Thanks for your interest in contributing to Lodestar. It's people like you that
 - :gear: [NodeJS](https://nodejs.org/) (LTS)
 - :toolbox: [Yarn](https://yarnpkg.com/)
 
+### MacOS Specifics
+
+When using MacOS, there are a couple of extra prerequisites that are required.
+
+- python
+- coreutils (e.g. via `brew install coreutils`)
```

Comment:
I used MacOS regularly for years but never intentionally installed these to start with lodestar (that package may be installed with git or other tool). 

Out of curiosity when did you encounter error for these in reference to lodestar?

Reply:
Maybe it's related to the node version? This might not happen if there are pre-builts available for blst